Reply method

Yutsa opened the pull request:
Here's the pull resquest for my reply method.

Yutsa committed:
Added 'reply' method.

Yutsa committed:
Added 'reply' method.

rhiever commented on TwitterFollowBot/__init__.py:
```diff
@@ -451,3 +451,16 @@ def send_tweet(self, message):
+            Count is the number of people to tweet to.
+        """
+
+        result = self.search_tweets(keyword, 2)
```
Please edit this line to take the `count` parameter instead of `2`.

rhiever commented:
This looks great. Thank you for the PR. I added a comment on one line that needs to be fixed before I can merge it.

I think it would also be helpful to keep a list of the tweets that have been responded to, so this `reply` method doesn't reply to the same tweets again and again. Thoughts?

Yutsa committed:
Reply method updated.

Yutsa commented:
Yeah I updated the method since I made the PR. It stores the id of the tweet it already replied in a file now and some more things.
The thing is I modified the code for my personal use so maybe there are some things that are not really needed.
I tried to keep only the necessary code.
I'll do a PR request and you'll just ask me to modify what you don't like.

alfons3992 commented:
hey man, i don't understand one thing. i put the source on the bottom of the original twitterfollowbot and created a file called "messages.txt" i put in that file: "hello, lol, ..." etc. but my bot doesn't reply to any tweets. also put a word in the "...." = keyword bracket and count=5. what am i missing?

mohandsedu0 commented:
https://github.com/rhiever/TwitterFollowBot/issues/148#issue-569621309

music-phobia commented:
Could anyone make a function to automatically DM to those who started following you?
